<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23599" started-at="2022-02-16T21:15:48 PST" finished-at="2022-02-16T21:16:12 PST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="23599" started-at="2022-02-16T21:15:48 PST" finished-at="2022-02-16T21:16:12 PST">
      <class name="VariousConcepts.CRMTest">
        <test-method status="PASS" signature="readConfig()[pri:0, instance:VariousConcepts.CRMTest@9a7504c]" name="readConfig" is-config="true" duration-ms="12" started-at="2022-02-16T21:15:48 PST" finished-at="2022-02-16T21:15:48 PST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readConfig -->
        <test-method status="PASS" signature="init()[pri:0, instance:VariousConcepts.CRMTest@9a7504c]" name="init" is-config="true" duration-ms="6354" started-at="2022-02-16T21:15:48 PST" finished-at="2022-02-16T21:15:55 PST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="loginTest()[pri:1, instance:VariousConcepts.CRMTest@9a7504c]" name="loginTest" duration-ms="8818" started-at="2022-02-16T21:15:55 PST" finished-at="2022-02-16T21:16:04 PST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:VariousConcepts.CRMTest@9a7504c]" name="tearDown" is-config="true" duration-ms="751" started-at="2022-02-16T21:16:04 PST" finished-at="2022-02-16T21:16:04 PST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="init()[pri:0, instance:VariousConcepts.CRMTest@9a7504c]" name="init" is-config="true" duration-ms="3385" started-at="2022-02-16T21:16:04 PST" finished-at="2022-02-16T21:16:08 PST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="addContact()[pri:2, instance:VariousConcepts.CRMTest@9a7504c]" name="addContact" duration-ms="3513" started-at="2022-02-16T21:16:08 PST" finished-at="2022-02-16T21:16:11 PST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Add Contact page not found expected [Add Contact] but found [Full Name*]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Add Contact page not found expected [Add Contact] but found [Full Name*]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at VariousConcepts.CRMTest.addContact(CRMTest.java:115)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addContact -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:VariousConcepts.CRMTest@9a7504c]" name="tearDown" is-config="true" duration-ms="718" started-at="2022-02-16T21:16:11 PST" finished-at="2022-02-16T21:16:12 PST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- VariousConcepts.CRMTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
